zio-http/src/main/scala/zhttp/http/Request.scala

```diff
@@ -22,21 +30,67 @@ trait Request extends HeaderExtension[Request] { self =>
       override def url: URL                           = u
       override def getHeaders: Headers                = h
       override def remoteAddress: Option[InetAddress] = self.remoteAddress
-      override private[zhttp] def getBodyAsByteBuf    = self.getBodyAsByteBuf
+      override def unsafeBody(
+        msg: (
+          UnsafeChannel,
+          UnsafeContent,
+        ) => Unit,
+      ): Unit = self.unsafeBody(msg)
     }
   }
 
+  def unsafeBody(msg: (UnsafeChannel, UnsafeContent) => Unit): Unit
+
   /**
    * Decodes the content of request as a Chunk of Bytes
    */
-  def getBody: Task[Chunk[Byte]] =
-    getBodyAsByteBuf.flatMap(buf => Task(Chunk.fromArray(ByteBufUtil.getBytes(buf))))
+  def getBody: Queue[ByteBuf] = ???
+//    for {
```

Comment:
I think this is not needed as we have a getBodyAsStream anyways, ZStream.effectAsyc uses a queue internally

Reply:
`getBody` is required and it should return a `Chunk[Byte]`